### ISSUE TITLE
Antlr Tool gives parse error with "line number:column number" pair that makes no sense.

### ISSUE THREAD

**Issue by kaby76** [mike-lischke/antlr-ng#20]
Grammar: [ArangoDbParser.g4](https://github.com/antlr/grammars-v4/blob/7f917add1199e8b050a7117814899ebab50a3c3b/aql/ArangoDbParser.g4)

antlr-ng commit: commit a5434282539d33f688a80f1994efb38cdf889ea9

Output:
```
ken@DESKTOP-DL44R7B:~/antlr-ng$ tsx cli/runner.ts -Dlanguage=TypeScript ~/grammars-v4/aql/Generated-Antlr4ng/ArangoDbParser.g4
error(50): ArangoDbParser.g4:21:59: syntax error: extraneous input '+' expecting RPAREN
ken@DESKTOP-DL44R7B:~/antlr-ng$
```
First, the file should parse. (I made sure the tool is called via `tsx`.) Second, the "line:column" pair is "21:59", which makes no sense because that's in the middle of a comment, and there are only 13 characters in [line 21](https://github.com/antlr/grammars-v4/blob/7f917add1199e8b050a7117814899ebab50a3c3b/aql/ArangoDbParser.g4#L21).

**Comment by mike-lischke** [comment]
That's probably coming from the artificial rule code ANTLR generates to resolve left recursive rules. Need to dig deeper...

**Comment by mike-lischke** [comment]
Looks like this was fixed by my patch 0a29ec5fc16e3057ec900a424f9c25f530ac6f77.
